app/views/orders/confirmation.html.erb

```diff
@@ -0,0 +1,47 @@
+<div class="row">
+  <div class="col-md-12 text-center">
+    <h1>ORDER CONFIRMATION</h1>
+    <h4>Mischief Managed. Your order has been submitted.</h4>
+    <h5>Total Cost: $<%= number_with_precision(@subtotal, :precision => 2, :delimiter => ',') %></h5>
+    <h5>Order Status: <%= @order.status %></h5>
+    <h5>Order Time: <%= @order.order_time.strftime("%m/%d/%Y %I:%M%p") %></h5>
+  </div>
+</div>
+<br>
+<br>
+<% @order.order_items.each do |order_item| %>
+  <div class="row">
+    <div class="col-md-2 col-md-offset-3">
+      <div class="cart-image-container"><%= link_to image_tag(order_item.product.photo_url,
```

Comment:
Nice job her with link_to and image tag integration. This is one of those things people take for granted but can take a while to get correct.